src/fetch/orderbook.py

```diff
@@ -45,7 +46,14 @@ class OrderbookFetcher:
     def _pg_engine(db_env: OrderbookEnv) -> Engine:
         """Returns a connection to postgres database"""
         load_dotenv()
-        db_url = os.environ[f"{db_env}_DB_URL"]
+        if "NETWORK" in os.environ:
+            db_url = (
+                os.environ[f"{db_env}_DB_URL"]
+                + "/"
+                + os.environ.get("NETWORK", "mainnet")
+            )
+        else:
+            db_url = os.environ[f"{db_env}_DB_URL"]
```

Comment:
This PR changes the meaning of the `PROD_DB_URL` and `BARN_DB_URL` env variables, and this `else` statement just ensures that it defaults to the "old" way, so that the tests won't fail. This should be removed once the secrets in the repo are edited.